src/App.css

```diff
@@ -39,7 +44,7 @@
   padding: 5px 0 20px 0;
 }
 
-tr {
+.btn btn-primary tr {
```

Comment:
See: https://www.w3schools.com/cssref/css_selectors.php

This selector applies styles to:

* all `tr` elements...
* inside a `btn-primary` element (which doesn't exist- `btn-primary` is a css class)...
* inside an element with `class=btn`

I'm not sure what your intention was here- let's chat about it when we meet.